modules/apps/questions/questions-web/src/main/resources/META-INF/resources/js/components/QuestionRow.es.js

```diff
@@ -55,9 +57,14 @@ export default function QuestionRow({
 				portraitURL: '',
 				userId: '0',
 		  };
+	const isRowSelected = rowSelected === question.friendlyUrlPath;
```

Comment:
Add Space and Reverse order

```suggestion

	const isRowSelected = question.friendlyUrlPath === rowSelected;
```